flow360/component/simulation/user_defined_dynamics/user_defined_dynamics.py

```diff
@@ -11,7 +11,41 @@
 
 
 class UserDefinedDynamic(Flow360BaseModel):
-    """:class:`UserDefinedDynamic` class for defining the user defined dynamics inputs."""
+    """
+    :class:`UserDefinedDynamic` class for defining the user defined dynamics inputs.
+
+    Example
+    -------
+
```

Comment:
Can you add link to where this comes from?

Reply:
Yes, add the link to the corresponding tutorial page